frontend/internal-packages/db/package.json

```diff
@@ -14,7 +14,7 @@
     "@biomejs/biome": "1.9.4",
     "@liam-hq/configs": "workspace:*",
     "eslint": "9.27.0",
-    "supabase": "2.23.1",
+    "supabase": "2.24.3",
```

Comment:
I faced an unexplained problem with differences between local and CI, but the update solved the problem.